UPGRADING.md

```diff
@@ -142,7 +155,22 @@ entry = datacatalog_v1.Entry()
 entry.type_ = datacatalog_v1.EntryType.FILESET
 ```
 
+The renamed attributes are:
+
+* `TagTemplateField.type` -> `TagTemplatedField.type_`
+* `ColumnSchema.type` -> `ColumnSchema.type_`
+* `Entry.type` -> `Entry.type_`
+
 ## Common Resource Path Helper Methods
 
+| Applicable previous versions |
+|:-----------------------------|
+| v2.0.0 or lower              |
```

Comment:
This is confusing to me. I believe 2.0 removed `location_path`, but 3.0 added `common_location_path` back.

Reply:
You mean is:
"Users of v2.0.0 need not to migrate (ie: from their point of view, it's just new feature rather than breaking change)"
right?

So maybe better to write "v1.0.0 or lower"?

Reply:
Yeah. I've made a suggested edit to clarify.